### DISCUSSION TITLE
wxhelper-3.9.2.23-V5版本 报错

### DISCUSSION THREAD

**Discussion by xushuyi888** [ttttupup/wxhelper#114] [Q&A]
C:\wxhelper\tool\injector>ConsoleInject.exe -i WeChat.exe -p C:\wxhelper\tool\injector\injector.dll -P 1888
Unknown option character `\x50'.

**Comment by ttttupup** [comment]
这个injector.dll 是注入dll不是被注入的dll。-P 指定端口需要指定的版本，3.9.2.23-V5 不支持指定端口。
